radix-engine/src/errors.rs

```diff
@@ -61,10 +61,10 @@ pub enum RuntimeError {
     IdAllocatorError(IdAllocatorError),
 
     /// Error when invoking an export.
-    InvokeError(Error),
+    InvokeError(),
```

Comment:
Can we remove the `()` if there is no arguments? This makes mapping of error much easier.

Reply:
Yup good point